Lib/test/test_grammar.py

```diff
@@ -297,7 +297,6 @@ def test_eval_input(self):
         # testlist ENDMARKER
         x = eval('1, 0 or 1')
 
-    @unittest.skip("TODO: RUSTPYTHON")
     def test_var_annot_basics(self):
         # all these should be allowed
         var1: int = 5
```

Comment:
It looks like this is failing in CI

Reply:
Has anyway to check a annotation assign in a function scope?
 [Annotations are only evaluated in a module or class.](https://github.com/python/cpython/blob/master/Python/compile.c#L5182)